apps/src/StudioApp.js

```diff
@@ -617,8 +634,10 @@ StudioApp.prototype.scaleLegacyShare = function () {
 
   var frameWidth = $(phoneFrameScreen).width();
   var scale = frameWidth / vizWidth;
-  applyTransformOrigin(vizContainer, 'left top');
-  applyTransformScale(vizContainer, 'scale(' + scale + ')');
+  if (scale !== 1) {
+    applyTransformOrigin(vizContainer, 'left top');
+    applyTransformScale(vizContainer, 'scale(' + scale + ')');
```

Comment:
Is this some sort of performance optimization?  Nit: It seems like it might be "simpler" to always do this, even when scale is 1 (in case something depends on these being set).